packages/flutter/test/material/outline_button_test.dart

```diff
@@ -412,11 +418,33 @@ void main() {
     gesture = await tester.startGesture(center);
     await tester.pump(); // Start the button fill animation.
     await tester.pump(const Duration(milliseconds: 200)); // Animation is complete.
-    checkPhysicalLayer(buttonElement, fillColor.withAlpha(0xFF));
+    _checkPhysicalLayer(buttonElement, fillColor.withAlpha(0xFF));
 
     // Tap gesture completes, button returns to its initial configuration.
     await gesture.up();
     await tester.pumpAndSettle();
-    checkPhysicalLayer(buttonElement, fillColor.withAlpha(0x00));
+    _checkPhysicalLayer(buttonElement, fillColor.withAlpha(0x00));
   });
 }
+
+PhysicalModelLayer _findPhysicalLayer(Element element) {
```

Comment:
these were just renamed and moved right? No functionality change?

Reply:
That's right. They were jammed at the top of main() with the wrong indent. We usually stick helpers like this at the end of test files and often (not always) use file-private names. That's all I did here.